src/components/pages/DetailsPage/DetailsPage.tsx

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import style from './DetailsPage.module.scss';
+import { useNavigate, useParams } from 'react-router-dom';
+import { ROUTER_PATHS, URL } from '../../../models/enums';
+import { ShowData } from '../../../models/interfaces';
+import { AiFillStar } from 'react-icons/ai';
+import { GoLinkExternal } from 'react-icons/go';
+import Spinner from '../../Spinner/Spinner';
+
+interface DetailsPageProps {
+  setIsCardItemsDarked: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const DetailsPage: React.FC<DetailsPageProps> = ({ setIsCardItemsDarked }) => {
+  const { id } = useParams();
+
+  const navigate = useNavigate();
+
+  const goBack = () => {
+    navigate(`${ROUTER_PATHS.MAIN}`);
+    setIsCardItemsDarked(false);
+  };
+
+  const noDate = 'no data to show';
+
+  const [isLoading, setIsLoading] = React.useState(false);
+  const [show, setShow] = React.useState({
+    id: 0,
+    image: '',
+    name: '',
+    rating: 0,
+    language: '',
+    summary: '',
+    genres: [noDate],
+    premiered: '',
+    ended: '',
+    officialSite: '',
+  });
+
+  const getShowById = async () => {
```

Comment:
Wrap this method in a useCallback hook, so then you won't need an eslint exception for dependency array on 81st line. The similar situation was described in the App.tsx component

Reply:
fixed